cli/config.go

```diff
@@ -138,6 +139,7 @@ func loadConfig(rootdir string, flags *pflag.FlagSet) (*viper.Viper, error) {
 		Output:           cfg.GetString("log.output"),
 		EnableStackTrace: cfg.GetBool("log.stacktrace"),
 		EnableSource:     cfg.GetBool("log.source"),
+		DisableColor:     cfg.GetBool("log.nocolor"),
```

Comment:
:grin:

Reply:
suggestion: The `// set default logging config` appears incorrect though - this is not setting the default, but instead takes from the CLI params?